tools/gn

```diff
@@ -340,8 +339,10 @@ def parse_args(args):
                       help='The IDE files to generate using GN. Use `gn gen help` and look for the --ide flag to' +
                       ' see supported IDEs. If this flag is not specified, a platform specific default is selected.')
 
-  parser.add_argument('--build-glfw-shell', dest='build_glfw_shell', default=False, action='store_true',
-                      help='Force building the GLFW shell on desktop platforms where it is not built by default.')
+  parser.add_argument('--build-glfw-shell', action='store_const', const=True,
```

Comment:
I thought argparse implicitly added a `no-` flag variant?

Reply:
I'm definitely not an argparse expert, but we're manually adding `no-` for `stripped` below already, and some quick searching turns up SO questions suggesting that it doesn't.

Reply:
I am not sure either, but if this pattern has been followed before, thats fine by me (I did not notice it in the initial review).